ChangeLog.md

```diff
@@ -23,6 +23,11 @@
 * Changed parameters 'BaseSizeTiB' and 'ExtendedCapacitySizeTiB' in cmdlet 'New-AzElasticSan' to be optional.
     - 'New-AzElasticSan'
 
+#### Az.EventHub 
```

Comment:
Not this one. Please move these to the `Upcoming Release` of `src/EventHub/EventHub/ChangeLog.md`